nijmegen/Issues/employeeData/description.xml

```diff
@@ -0,0 +1,45 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Employee data</en>
+		<nl>Demo Personeelsgegevens</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Employee data</en>
+		<nl>Demo Personeelsgegevens</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>nijmegen</IssuerID>
+	<CredentialID>employeeData</CredentialID>
+	<Description>
+		<en>Your employee data, from Gemeente Nijmegen</en>
+		<nl>Uw personeelsgegevens, van Gemeente Nijmegen</nl>
+	</Description>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.nijmegen.employeeData.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.nijmegen.employeeData.html</nl>
+	</IssueURL>
+	<ShouldBeSingleton>true</ShouldBeSingleton>
+
+	<Attributes>
+		<Attribute id="email" displayIndex="0">
+			<Name>
+				<en>Email address</en>
+				<nl>E-mailadres</nl>
+			</Name>
+			<Description>
+				<en>Your Gemeente Nijmegen email address</en>
+				<nl>Uw Gemeente Nijmegen e-mailadres</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="works-for-gemeente-nijmegen" displayIndex="1">
```

Comment:
Usually we have lowerCamelCase or alllowercase for attribute ids. It's also fine this way, but that might be something to consider.

Reply:
Dat is inderdaad ook beter in lijn met de naam van het kaartje.